dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: OpenAMP (RPMsg with static VRINGs) backend
+
+compatible: "zephyr,ipc-openamp-static-vrings"
+
+include: base.yaml
+
+properties:
```

Comment:
WOW, this is a really great new, thank you for pointing this out. I'll definitely try to align the two works.

Reply:
The linux maintainers still have to be convinced, so this will probably change a bit.  The name of the `virtio` node might change, as well as the `virtio,id` fields and the `compatible` value.
So perhaps don't lost your time to align both. Updating the `shm` to `memory-region` is probably enough for now.

Reply:
ACK